wiki/Tournaments/ANZT/2019_Summer/en.md

```diff
@@ -0,0 +1,112 @@
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-12-9/2018-12-30 |
+| Group Stage | 2019-01-11/2019-02-10 |
+| Round of 16 | 2019-02-15/2019-02-17 |
+| Quarterfinals | 2019-02-22/2019-02-24 |
+| Semifinals | 2019-03-01/2019-03-03 |
+| Grand Finals | 2019-03-08/2019-03-10 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 4 months of osu!supporter tag, unique profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month of osu!supporter tag |
+
+## Organisation
+
+| Position | Member(s) |
+| :-- | :-- |
+| Management | ![][flag_NZ] [Diony](https://osu.ppy.sh/users/4294475), ![][flag_AU] [EmeraldStar82](https://osu.ppy.sh/users/3772135) |
+| Scheduling | ![][flag_AU] [EmeraldStar82](https://osu.ppy.sh/users/3772135),  |
+| Commentators | ![][flag_AU] [abolaspany](https://osu.ppy.sh/users/4261853), ![][flag_AU] [awesometa](https://osu.ppy.sh/users/7114425), ![][flag_NZ] [Monster Bait](https://osu.ppy.sh/u/6890546), ![][flag_AU] [Monk Gyatso](https://osu.ppy.sh/u/4012086), ![][flag_AU] [Arbusion](https://osu.ppy.sh/users/11469447), ![][flag_AU] [HolyJesus](https://osu.ppy.sh/users/7785655), ![][flag_AU] [Aus](https://osu.ppy.sh/users/6541404), ![][flag_AU] [Takagaki](https://osu.ppy.sh/users/3698691), ![][flag_AU] [pewdekz](https://osu.ppy.sh/users/5754859) |
+| Streamers | ![][flag_AU] [pewdekz](https://osu.ppy.sh/users/5754859), ![][flag_VN] [steve_04_](https://osu.ppy.sh/users/10852911), ![][flag_AU] [TheOmyNomy](https://osu.ppy.sh/users/4241054) |
+| Referees | ![][flag_AU] [Illinon](https://osu.ppy.sh/users/8306102), ![][flag_AU] [Soulskyeflare](https://osu.ppy.sh/users/8264319), ![][flag_AU] [awesometa](https://osu.ppy.sh/users/7114425), ![][flag_PH] [ggnakacarlz](https://osu.ppy.sh/users/4075421), ![][flag_NZ] [phophicles](https://osu.ppy.sh/users/9373724), ![][flag_VN] [steve_04_](https://osu.ppy.sh/users/10852911), ![][flag_NZ] [NutAndGo](https://osu.ppy.sh/users/7740118), ![][flag_KR] [silverlightcs](https://osu.ppy.sh/users/9862564), ![][flag_NZ] [Div](https://osu.ppy.sh/users/3751116), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811) |
+| Mappool Selectors | ![][flag_JP] [Delis](https://osu.ppy.sh/users/1603923), ![][flag_JP] [Angel Arrow](https://osu.ppy.sh/users/1970239), ![][flag_MV] [HypebeastHijabi](https://osu.ppy.sh/users/1786945) |
+| Graphics Design | ![][flag_AU] [abolaspany](https://osu.ppy.sh/users/4261853) |
+
+## Links
+
+- [Discussion thread](/community/forums/topics/839284)
```

Comment:
```suggestion
- [Discussion thread](https://osu.ppy.sh/community/forums/topics/839284)
```

Reply:
I think the original is correct here looking at "Other osu! web pages" under Links on the styling criteria.

Reply:
we'll update that soon, but this is now what we're enforcing. everything besides /wiki should have the hostname